FateGrandAutomata/SuperUser.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Java.Lang;
+using Exception = Java.Lang.Exception;
+
+namespace FateGrandAutomata
+{
+    public class SuperUser : IDisposable
+    {
+        readonly Process _superUser;
+        readonly StreamWriter _superUserStreamWriter;
+
+        public SuperUser()
+        {
+            try
+            {
+                _superUser = Runtime.GetRuntime().Exec("su", null, null);
+                _superUserStreamWriter = new StreamWriter(_superUser.OutputStream, Encoding.ASCII);
+            }
+            catch (Exception e)
+            {
+                throw new System.Exception("Failed to get Root permission", e);
+            }
+        }
+
+        public void SendCommand(string Command)
+        {
+            _superUserStreamWriter.WriteLine(Command);
+            _superUserStreamWriter.Flush();
+
+            // Wait
+            // https://stackoverflow.com/a/16160785/5377194
+            {
```

Comment:
Any reason for `{`?

Reply:
No, it was just there for attribution as a block. I'll add it to separate function.
I'm refactoring the code a bit and adding the root gestures. Will be done in a few minutes.